concert/src/main/resources/application.yml

```
@@ -24,4 +24,4 @@ server:
 eureka:
   client:
     service-url:
-      defaultZone: http://${EUREKA_HOST:localhost}:${EUREKA_PORT:8761}/eureka
\ No newline at end of file
+      defaultZone: http://${EUREKA_HOST:localhost}:${EUREKA_PORT:18080}/eureka
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)